plugin/src/test/java/com/google/errorprone/xplat/checker/JodaTimeLocalTest.java

```diff
@@ -0,0 +1,359 @@
+                + " ldt.getHourOfDay(),"
+                + " ldt.getMinuteOfHour(), ldt.getSecondOfMinute(), ldt.getMillisOfSecond(),"
+                + " DateTimeZone.forID(\"America / New_York\"))",
+            "    .toDateTime(DateTimeZone.forID(\"America / Los_Angeles\"));",
+            "  }",
+            "}")
+        .doTest();
+  }
+
+  // LocalDateTime -> DateTime without DateTimeZone
+  @Test
+  public void refactorLocalDateTime2() {
+    BugCheckerRefactoringTestHelper.newInstance(new JodaTimeLocal(), getClass())
+        .addInputLines("Test.java",
+            "import org.joda.time.DateTime;",
+            "import org.joda.time.DateTimeZone;",
+            "import org.joda.time.LocalDateTime;",
+            "class Test {",
+            "  private DateTime badLocalDateTimeUse() {",
+            "    LocalDateTime ldt = new LocalDateTime(2020, 6, 2, 8, 0, 0, 0);",
+            "    return ldt.toDateTime()",
+            "      .toDateTime(DateTimeZone.forID(\"America / Los_Angeles\"));",
+            "  }",
+            "}")
+        .addOutputLines("Test.java",
+            "import org.joda.time.DateTime;",
+            "import org.joda.time.DateTimeZone;",
+            "import org.joda.time.LocalDateTime;",
+            "class Test {",
+            "  private DateTime badLocalDateTimeUse() {",
+            "    LocalDateTime ldt = new LocalDateTime(2020, 6, 2, 8, 0, 0, 0);",
+            "    return new DateTime(ldt.getYear(), ldt.getMonthOfYear(), ldt.getDayOfYear(),"
+                + " ldt.getHourOfDay(),"
+                + " ldt.getMinuteOfHour(), ldt.getSecondOfMinute(), ldt.getMillisOfSecond(),"
+                + " DateTimeZone.getDefault())",
+            "      .toDateTime(DateTimeZone.forID(\"America / Los_Angeles\"));",
+            "  }",
+            "}")
+        .doTest();
+  }
+
+  // LocalTime -> DateTime with DateTimeZone
+  @Test
+  public void refactorLocalTime() {
+    BugCheckerRefactoringTestHelper.newInstance(new JodaTimeLocal(), getClass())
+        .addInputLines("Test.java",
+            "import org.joda.time.DateTime;",
+            "import org.joda.time.DateTimeZone;",
+            "import org.joda.time.LocalTime;",
+            "class Test {",
+            "  private DateTime badLocalTimeUse() {",
+            "    LocalTime lt = new LocalTime(8, 0, 0, 0);",
+            "    return lt.toDateTimeToday(DateTimeZone.forID(\"America / New_York\"))",
+            "      .toDateTime(DateTimeZone.forID(\"America / Los_Angeles\"));",
+            "  }",
+            "}")
+        .addOutputLines("Test.java",
+            "import org.joda.time.DateTime;",
+            "import org.joda.time.DateTimeZone;",
+            "import org.joda.time.LocalTime;",
+            "class Test {",
+            "  private DateTime badLocalTimeUse() {",
+            "    LocalTime lt = new LocalTime(8, 0, 0, 0);",
+            "    return new DateTime().now(DateTimeZone.forID(\"America / New_York\"))"
```

Comment:
See my earlier comment in `JodaTimeLocal.java`: this should be `new DateTime().toDateTime(...)`.

Reply:
Fixed